code/ui/alivehud/c4/C4ArmControl.cs

```diff
@@ -0,0 +1,224 @@
+using System;
+using System.Collections.Generic;
+
+using Sandbox;
+using Sandbox.UI;
+using Sandbox.UI.Construct;
+
+using TTTReborn.Items;
+using TTTReborn.Player;
+using TTTReborn.UI.Construct;
```

Comment:
We don't use this. Construct is a sbox thing most of all, that's why we created even ui component construct things inside the sandbox `Construct` namespace.